scipy/stats/morestats.py

```diff
@@ -1082,9 +1082,9 @@ def anderson(x, dist='norm'):
            pp. 591-595.
 
     """
-    if dist not in ['norm', 'expon', 'gumbel', 'extreme1', 'logistic']:
+    if dist not in ['norm', 'expon', 'gumbel_l', 'gumbel_r', 'extreme1', 'logistic']:
```

Comment:
looks like ``gumbel`` should be kept here as an alias of ``gumbel_l``, to not break backwards compatibility? 

It doesn't need to be in the docstring.